public/apps/configuration/utils/resource-validation-util.tsx

```diff
@@ -34,7 +32,7 @@ export function validateResourceName(resourceType: string, resourceName: string)
     errors.push(VALID_LENGTH_HELP_TEXT(resourceType));
   }
 
-  if (!isResourceNameValid(resourceName)) {
+  if (!isValidResourceName(resourceName)) {
```

Comment:
did you tested this using browser? not sure the regex using unicode can work on browsers

Reply:
![image](https://user-images.githubusercontent.com/63078162/97366201-2581ec80-1864-11eb-800d-
5e22a198a89c.png)
Yep it is easy for chrome in console.

Reply:
Work with 
Chrome version 85.0.4183.121
Safari version 13.1.2
But not work with 
Firefox 68.11

I guess I need to revert the removal of XRegExp

Reply:
Update, though it doesn't work in Firefox console, it actually work on page. It has been translated to unicode range:
![image](https://user-images.githubusercontent.com/63078162/97368267-c2925480-1867-11eb-9dca-6d585efb8b49.png)